### DISCUSSION TITLE
How do I set Sheriff kill cooldown longer than 40 seconds?

### DISCUSSION THREAD

**Discussion by PieCreeper12** [Woodi-dev/Among-Us-Sheriff-Mod#115] [Q&A]
The mod only lets me set the sheriff kill cooldown up to 40. How do I make it so it is longer?

**Comment by Woodi-dev** [comment]
You cant. The max Cooldown is set to 40. If you wanna change it you need to edit this behaviour at GameOptionsMenuPatch and compile the mod.
